include/evmc/evmc.h

```diff
@@ -70,58 +70,58 @@ enum evmc_flags
  */
 struct evmc_message
 {
+    /** The kind of the call. For zero-depth calls ::EVMC_CALL SHOULD be used. */
```

Comment:
While this isn't changed in this PR, but I think this comment is invalid, it can be create even for depth 0.